src/sentry/conf/server.py

```diff
@@ -740,6 +740,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
         routing_key="dynamicsampling",
     ),
     Queue("auto_enable_codecov", routing_key="auto_enable_codecov"),
+    Queue("weekly_escalating_forecast", routing_key="weekly_escalating_forecast"),
```

Comment:
Just as a reminder, you'll need to ask ops to provision workers for the new queue here before this will be processed

Reply:
Thank you, I have the ticket [here](https://getsentry.atlassian.net/browse/OPS-3170)